CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
 -----------
 
 ### Internals
-* None.
+* Realm::write_copy_without_client_file_id and Realm::write_copy are merged.
```

Comment:
Does it make sense to add another sentence here explaining what the result of that merge is so that the SDKs know what to expect?
Since the new `write_copy` does not work like the old `write_copy` having a hint here might help other SDKs not being thrown off by it. Especially if they haven't adapted `write_copy_without_client_file_id` in the meantime this basically means, they'll have (potentially) a breaking change in `write_copy`.

Reply:
I will add a comment